operator/cmd/mesh/testdata/manifest-generate/data-snapshot/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -197,6 +197,14 @@ template: |
           {{- end}}
         {{- end}}
         ]
+    - name: ISTIO_META_APP_CONTAINERS
```

Comment:
not sure how this change made it into this PR?

Reply:
the injections goldens were made against a more recent version of the charts because prior to this pr they are still being generated against the live charts. after this pr, they will pointing to the snapshot so they will not need to be refreshed unless the logic changes in a way that affects output.